src/components/ha-selector/ha-selector-select.ts

```diff
@@ -118,9 +118,11 @@ export class HaSelectSelector extends LitElement {
           .disabled=${this.disabled}
           .required=${this.required && !value.length}
           .value=${this._filter}
+          .items=${options}
```

Comment:
does it need `items` when it already has `filteredItems`?

Reply:
Looks like filteredItems doesn't work when items is not set.

Reply:
Oh I found the issue. The `_filterChanged` is using `items`.